pkg/manager/member/tidb_member_manager.go

```diff
@@ -387,12 +387,13 @@ func (tmm *tidbMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, se
 			Health: health,
 		}
 		oldTidbMember, exist := tc.Status.TiDB.Members[name]
+
+		newTidbMember.LastTransitionTime = metav1.Now()
```

Comment:
This looks like a bugfix instead of just a readability change?

Reply:
Not a bugfix, the logic is still the same as before